tests/test_market.py

```diff
@@ -0,0 +1,95 @@
+"""
+Testing for the ElfPy package modules
+"""
+
+# pylint: disable=too-many-instance-attributes
+# pylint: disable=too-many-locals
+# pylint: disable=attribute-defined-outside-init
+# pylint: disable=duplicate-code
+
+import unittest
+import logging
+
+import numpy as np
+
+import elfpy.utils.outputs as output_utils  # utilities for file outputs
+import elfpy.utils.testing as test_utils  # utilities for testing
+
+
+class BaseMarketTest(unittest.TestCase):
+    """Generic Parameter Test class"""
+
+    def run_base_market_test_one_timestep(
+        self,
+        agent_policies,
+        config_file="config/example_config.toml",
+        delete_logs=True,
+    ):
+        """Create base structure for future tests"""
+        output_utils.setup_logging(log_filename=".logging/test_trades.log", log_level=logging.DEBUG)
+        # load default config
+        override_dict = {
+            "pricing_model_name": "Yieldspace",
+            "target_liquidity": 10e6,
+            "fee_percent": 0.1,
+            "target_pool_apr": 0.05,
+            "vault_apr": {"type": "constant", "value": 0.05},
+            # minimal simulation steps, we only care to investigate the first day's trades
+            "num_trading_days": 10,
```

Comment:
running for 10 days with logging level set to DEBUG could take some time. I recommend lowering this to the minimum number of days that is needed to accomplish the testing task (probably 3?)